.travis.yml

```diff
@@ -1,10 +1,10 @@
 language: node_js
 
 node_js:
-  - '0.10'
-  - '0.12'
-  - '4.2'
-  - 'stable'
+  - 6
+  - 8
+  - lts/*
```

Comment:
Just add `6`, `8`, `lts/*`, leave the rest untouched, we will remove old node support in the next major, but until then this would be a breaking change.

Reply:
Reapplied those versions, but seems to be an issue with the 4.2 pipleline job - https://travis-ci.org/SassDoc/scss-comment-parser/jobs/528927921

Can't see it being related to the changes in the PR, can't even get yarn to run using that version locally.

Worth trying 4.8 that's mentioned in the warning? Same major version so shouldn't be considered breaking?